### ISSUE TITLE
Color Contrast Example - Build Error

### ISSUE THREAD

**Issue by retiutut** [BrainJS/brain.js#524]
![A GIF or MEME to give some spice of the internet](https://media.giphy.com/media/U6Fxnc2jTlBh2GKCTU/giphy.gif)

## *What* is wrong?

`npm run build` fails with slightly modified color contrast example

## *Where* does it happen?

It happens when trying to run `npm run build` on my Mac running NodeJS LTS.

## *How* do we replicate the issue?

Build script in package.json: `"build": "cross-env BABEL_ENV=production babel src --out-dir lib",`

```
src/index.js -> lib/index.js
TypeError: src/neuralNetTest.js: Cannot read property 'contexts' of null
```

As you can see, I have an index.js that is able to build, but the example code contained in `src/neuralNetTest.js` shows a TypeError.

## *How* important is this (1-5)?

4-5

## Expected behavior (i.e. solution)

Build without errors

## Other Comments
```
const brain = require('brain.js');

const net = new brain.NeuralNetwork();

const trainNN = () => {
  net.train([{
    input: {
      r: 0.03,
      g: 0.7,
      b: 0.5
    },
    output: {
      black: 1
    }
  },
  {
    input: {
      r: 0.16,
      g: 0.09,
      b: 0.2
    },
    output: {
      white: 1
    }
  },
  {
    input: {
      r: 0.5,
      g: 0.5,
      b: 1.0
    },
    output: {
      white: 1
    }
  }
  ]);
};

const runNN = (red, green, blue) => {
  trainNN();

  const output = net.run({
    r: red,
    g: green,
    b: blue
  }); // { white: 0.99, black: 0.002 }

  return output;
};

export { trainNN, runNN };
```

**Comment by retiutut** [comment]
`.babelrc`:

```
{
  "env": {
    "development": {
      "presets": ["env"],
      "plugins": [
        "add-module-exports"
      ]
    },
    "production": {
      "presets": ["env", "minify"],
      "plugins": [
        "add-module-exports"
      ]
    }
  }
}
```

**Comment by retiutut** [comment]
Temporary fix: remove "minify" from production build preset.

**Comment by retiutut** [comment]
@robertleeplummerjr

**Comment by mubaidr** [comment]
This is most likely caused by babel `config`. Can you please test the pre-built versions provided? 

Also please specify nodejs/browser version.

**Comment by retiutut** [comment]
I was using Node version 10.16.3. I updated to 14.1.0, and the error is still happening.

I'm not sure what the point of using pre-built is, since I am trying to build the examples myself.

Here is the error, again, caused by "minify" option when building using Babel (latest version):

```
> cross-env BABEL_ENV=production babel src --out-dir lib

src/index.js -> lib/index.js
TypeError: src/neuralNetTest.js: Cannot read property 'contexts' of null
    at NodePath._getQueueContexts (/Users/Richard/Documents/GitHub/*/node_modules/babel-traverse/lib/path/context.js:278:21)
    at NodePath._containerInsert (/Users/Richard/Documents/GitHub/*/node_modules/babel-traverse/lib/path/modification.js:94:23)
    at NodePath._containerInsertBefore (/Users/Richard/Documents/GitHub/*/node_modules/babel-traverse/lib/path/modification.js:137:15)
    at NodePath.insertBefore

etc.
```

**Comment by retiutut** [comment]
Even without changing the Color Contrast example at all, still fails to build using Babel with "minify" enabled.

**Comment by mubaidr** [comment]
Please share `babel` config and other info. I will try to reproduce this issue asap.

**Comment by retiutut** [comment]
https://github.com/BrainJS/brain.js/issues/524#issuecomment-609465812

**Comment by mubaidr** [comment]
`babel-preset-env` and `babel-minify` does not seems to be compatible: https://github.com/babel/minify/issues/664

Also since `babel-minify` in in beta, i would recommend using other tools (parcel, rollup etc) to minify output.
